articles/azure-node-health-checks/reviewing-results.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Interpreting results from Azure Node Health Check
+description: Interpreting results from Azure Node Health Checks (AzNHC) to validation.
+ms.service: azure-node-health-checks
+ms.topic: how to 
+author: rafaelsalas
+ms.author: rafaelsalas
+ms.date: 04/15/2024
+---
+
+# Interpreting Azure Node Health Check Results
+AzNHC prints results to both terminal and it's health log. The error messages produced by AzNHC gives are useful for determining the failed components. The following will guide you through understanding what a successful/failed test output looks like.
+
+## Test Results
+
+### Passing Tests
+By default AzNHC will run each test specified in the conf file to completion unless a failure is detected.
+
+Passing tests have this format:
+  ```SUCCESS:  nhc:  Health check passed:  <Test function executed>: <Test result message>```
+
+### Failed Tests
+If AzNHC encounters a failure it will exit and report the failure. If you would like to run all tests regardless of any failure you can use the "--all" flag when running.
+
+Failed tests have this format:
+  ```ERROR:  nhc:  Health check failed:  <Test function executed>: <Test result message>. FaultCode: <NHC Fault code>```
```

Comment:
It would be super helpful if the error output had an optional row with a mitigation suggestion where possible mitigations are an enum (something like:  [reboot, reimage, report/unrecoverable, unknown] so that NHC test writers can indicate to the user how to deal with the failure (if known)

Reply:
We would have to track this request on the [AzNHC repo](https://github.com/Azure/azurehpc-health-checks). I believe we could add it though.